### PR TITLE
Resetting to origin pod, except for an iOS7 check.

### PR DESCRIPTION
**Pull request by cwshevlin**

### CODE REVIEW

File: TPKeyboardAvoiding/UIScrollView+TPKeyboardAvoidingAdditions.m [hunk 1]
```diff
@@ -42,6 +42,9 @@ - (TPKeyboardAvoidingState*)keyboardAvoidingState {
 }
 
 - (void)TPKeyboardAvoiding_keyboardWillShow:(NSNotification*)notification {
+    if (![[NSProcessInfo processInfo] respondsToSelector:@selector(isOperatingSystemAtLeastVersion:)]){
```
Comment (timoman) [inline TPKeyboardAvoiding/UIScrollView+TPKeyboardAvoidingAdditions.m:46]: Comment pleaseee (this isn't immediately clear as an iOS check for those unfamiliar with `isOperatingSystemAtLeastVersion`)

**Comment by thmartin** [comment]
add the other check back in too.

**Comment by cwshevlin** [comment]
Updated.

**Comment by timoman** [comment]
:shipit:

**Comment by thmartin** [comment]
:shipit:

**Pull request merged**